### PR TITLE
add podman-compose 0.1.5

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes @conda-forge/help-python ready for review!

File: recipes/podman-compose/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - python -m pip check
+    - podman-compose --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/containers/podman-compose
+  summary: A script to run docker-compose.yml using podman
+  license: GPL-2.0-or-later
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/podman-compose/meta.yaml:43]: I opened a PR to include the license file in the source distribution:
https://github.com/containers/podman-compose/pull/229

Maybe add a comment above the license file line as a reminder to check if it's included in a future release:

```yaml
...
# License file pulled in from separate source. See: https://github.com/containers/podman-compose/pull/229
license_file: LICENSE
```

**Review by synapticarbors — COMMENTED** [review]
Looks good. I just had a small suggestion.

**Comment by bollwyvl** [comment]
Cool you beat me to it... Was also going to pr a python_requires...

File: recipes/podman-compose/meta.yaml (hunk 1 above)
Comment (synapticarbors) [reply recipes/podman-compose/meta.yaml:44]: Thanks -- I'll merge as soon as the CI finishes.

**Pull request merged**